tavern/tomes/file_list/main.eldritch

```diff
@@ -1,17 +1,115 @@
+usernfo = sys.get_user()
+
+name = {
+    "Directory": "Dir",
+    "File": "File",
+    "Link": "Link"
+}
+
+SEP = "/"
+if sys.get_os().get("platform", "") == "PLATFORM_WINDOWS":
+    SEP = "\\"
+
+def can_read(f):
+    """Return true if the user can read this dir/file
+    """
+
+    # Until we get permissions on windows, just go ahead and try to read
```

Comment:
@hulto What do you think about this? better to try and fail then not try, right?

Reply:
Yeah for now just send it 🤷